tests/wpt/meta/content-security-policy/securitypolicyviolation/targeting.html.ini

```diff
@@ -4,13 +4,10 @@
     expected: NOTRUN
 
   [Inline violations target the right element.]
-    expected: TIMEOUT
+    expected: FAIL
```

Comment:
There are a lot of different test cases here. We pass the first one, but we fail the second one (inline event handler). I don't think the logic is incorrect per se (given all the other tests started passing), but to get these things correct we do need to investigate further. Therefore, I don't think this test is representative for the correctness of this patch, even though it is the exact WPT test that is supposed to verify that.